hack/istio/tempo/install-tempo-env.sh

```diff
@@ -80,6 +80,23 @@ done
 
 SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
 
+if [ "${CLIENT_EXE}" == "" ]; then
+  CLIENT_EXE=`which "${CLIENT_EXE_NAME}"`
+  if [ "$?" = "0" ]; then
+    echo "The cluster client executable is found here: ${CLIENT_EXE}"
+  else
+    echo "ERROR: You must install the cluster client ${CLIENT_EXE_NAME} in your PATH before you can continue."
+    exit 1
+  fi
+fi
+
+IS_OPENSHIFT="false"
+if [[ "${CLIENT_EXE}" = *"oc" ]]; then
+  IS_OPENSHIFT="true"
+fi
```

Comment:
I recently changed our other hack script to be more specific when determining if the server is OpenShift or not. 

We should consider doing this in our hack scripts now:

https://github.com/kiali/kiali/blob/master/hack/istio/install-istio-via-istioctl.sh#L281-L287

```suggestion
if ${CLIENT_EXE} api-versions | grep --quiet "route.openshift.io"; then
  IS_OPENSHIFT="true"
  echo "You are connecting to an OpenShift cluster"
else
  IS_OPENSHIFT="false"
  echo "You are connecting to a (non-OpenShift) Kubernetes cluster"
fi
```